lua/neogen/generator.lua

```diff
@@ -38,15 +40,60 @@ local function todo_text(type)
     })[type] or todo["description"]
 end
 
-local function get_parent_node(filetype, typ, language)
+local function populate_filetype(filetype, language)
```

Comment:
Can you annotate this function, describing what is it doing ? (protip: use neogen for neogen annotations)